ci/copy-unix-release-artifacts.sh

```diff
@@ -83,5 +87,14 @@ if [[ "$NAME" == "linux" ]]; then
     cp bazel-bin/runtime-components/non-repudiation-client/libnon-repudiation-client-src.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CLIENT_SRC
     cp bazel-bin/runtime-components/non-repudiation-client/non-repudiation-client_javadoc.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CLIENT_DOC
 
-fi
+    bazel build //compiler/damlc:damlc-dist
+    cp bazel-bin/compiler/damlc/damlc-dist.tar.gz $OUTPUT_DIR/split-release/damlc-$RELEASE_TAG.tar.gz
+
+    mkdir -p $OUTPUT_DIR/split-release/daml-libs/daml-script
+    bazel build //daml-script/daml:daml-script-dars
+    cp bazel-bin/daml-script/daml/*.dar $OUTPUT_DIR/split-release/daml-libs/daml-script/
```

Comment:
Will those include version numbers too? (Should they?)

Reply:
I considered it but in the end it seemed nice to keep the name they have in the SDK tarball. That way you can in theory make `daml-script` as a dependency work by putting it in the right directory and set the `DAML_SDK` env var even if you just have those dars and not the rest of the SDK. We do include the version in the url.

Reply:
That makes sense, thanks for explaining.